### PR TITLE
Adds forensic scanner to R&D.

### PR DESCRIPTION
**Pull request by drynwyn**
Player poll located at http://www.ss13.eu/phpbb/viewtopic.php?f=48&t=4414&p=136697#p136697

### CODE REVIEW

**Comment by drynwyn** [comment]
astftglha adfa i fatfingered something it seems

File: code/modules/research/designs.dm [hunk 1]
```diff
@@ -1869,6 +1869,15 @@ datum/design/borg_syndicate_module
 //////////////////Misc///////////////////
 /////////////////////////////////////////
 
+datum/design/detective_scanner
+	name = "forensic scanner"
+	desc = "Used to remotely scan objects and biomass for DNA and fingerprints. Can print a report of the findings."
+	id = "forensicscanner"
+	req_tech = list("materials" = 3, "magnets" = 5)
+	build_type = PROTOLATHE
+	materilas = list("$metal" = 3000, "$glass" = 3000)
```
Comment (Petethegoat) [inline code/modules/research/designs.dm]: >materilas

**Comment by drynwyn** [comment]
Fod damnit.

**Comment by drynwyn** [comment]
On iPad at present- will fix this evening.

**Comment by ghost** [comment]
way too easy to make imo, and with no lockbox it kind of takes away something special from the detective, especially considering he gets an extra forensic scanner anyway and they are rarely both lost.

**Comment by akenworthy87** [comment]
As I mentioned, not needed at the moment.

And if I get my sec overhaul in they'll be four-six scanners on the station at round start

**Comment by Atskadan** [comment]
please lockbox with hos requirement

**Comment by Atskadan** [comment]
> please lockbox with hos requirement

**Comment by drynwyn** [comment]
Lockboxes universally require Armory access. I can lockbox it, but it will be openable by anyone with Armory.

**Comment by Atskadan** [comment]
thats ok

**Pull request merged**